src/sources/file.rs

```diff
@@ -250,6 +250,12 @@ pub struct FileConfig {
     #[configurable(derived)]
     #[serde(default)]
     internal_metrics: FileInternalMetricsConfig,
+
+    /// How long to keep an open handle to a rotated log file.
+    #[serde_as(as = "serde_with::DurationMilliSeconds<u64>")]
+    #[configurable(metadata(docs::type_unit = "milliseconds"))]
+    #[serde(default = "default_rotate_wait_ms")]
+    pub rotate_wait_ms: Duration,
```

Comment:
For here and in the Kubernetes Logs source, `rotate_wait_secs` seems more appropriate than `rotate_wait_ms` as my understanding is that a reasonable value for this option is greater than 1 second. What do you think?